src/components/common/SearchFiltersWrapperV2/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useMemo } from 'react';
+import { Box, Grid } from '@mui/material';
+import { PillList, PillListItem, Textfield } from '@terraware/web-components';
+import strings from 'src/strings';
+import { SearchNodePayload } from 'src/types/Search';
+import { FilterField } from 'src/components/common/FilterGroup';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import IconFilters from './IconFilters';
+import FeaturedFilters from './FeaturedFilters';
+
+export type SearchInputProps = {
+  search: string;
+  onSearch: (search: string) => void;
+};
+
+export type FilterConfig = {
```

Comment:
should probably move this into IconFilters and let that be unaware of this file, there's a mutual dependence right now

Reply:
It is shared intentionally, the idea though is that you can show the same filter config in two different ways, "featured" or within the filter icon popover. A common interface with two implementations

Reply:
Got it 👍, that makes sense.